core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-background-columns/template-builder-background-columns.component.scss

```diff
@@ -2,13 +2,10 @@
 
 $columns-padding: 3.2rem;
 
-.background-columns {
-    height: 100%;
-    width: 100%;
+:host {
     grid-template-columns: repeat(12, minmax(0, 1fr));
     display: grid;
-    padding: 0 $columns-padding 0 $columns-padding;
-    position: absolute;
+    gap: $spacing-3;
```

Comment:
You need to get this from the const you created and you can pass it dynamically to the host using HostBinding.